manage.py

```diff
@@ -65,8 +67,13 @@ def test(path=None):
 
 @cli.command('seed_db')
 def seed_db():
-    """Seed the database with a set of users and flight schedules
```

Comment:
Lets tweak this a bit more by creating two functions `seed_db` and `seed_db_example`:

- `seed_db`: this will just drop and recreate the database, import the commands, and create one admin user and one non-admin user
- `seed_db_example`: this will do the same as `seed_db` but then also insert a flight schedule and some example passover times